stix_shifter_modules/synchronous_dummy/stix_transmission/synchronous_dummy_api_client.py

```diff
@@ -13,19 +12,21 @@ def __init__(self, connection, configuration):
         #                             connection.get('cert', None),
         #                             headers,
         #                             cert_verify=connection.get('cert_verify', 'True')
-        #
+        #                             )
 
         # Placeholder client to allow dummy transmission calls.
-        # Remove when implementing data source API client.                            )
+        # Remove when implementing data source API client.
         self.client = "data source API client"
 
-    def ping_box(self):
+    def ping_data_source(self):
         # Pings the data source
-        return {"code": 200, "results": "Was able to hit the data source"}
-
-    def run_search(self, query_expression, offset=None, length=None):
-        # headers = dict()
-        # return self.client.call_api(endpoint, 'GET', headers, urldata=data)
+        return {"code": 200, "success": True, "results": "Was able to hit the data source"}
```

Comment:
shouldn't we be consistent with the return object that we currently return? Currently for any successful call we returns objects like this-
ping-
`{'success': True}`
query/search-
`{'success': True, 'search_id': '78578351-6f49-4fc2-89ce-5be12dd36383'}`
 status:
`{'success': True, 'status': 'COMPLETED', 'progress': 100}`
results:
`{'success': True, 'data': []}`

Reply:
a good point, fixed that, added error mapping.

(virtualenv) [yurii@oc5337804250 stix-shifter]$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' ping
{'success': True}
(virtualenv) [yurii@oc5337804250 stix-shifter]$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' query 'query1'
{'success': True, 'search_id': 'query1'}
(virtualenv) [yurii@oc5337804250 stix-shifter]$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' status 'query1'
{'success': True, 'status': 'COMPLETED', 'progress': 100}
(virtualenv) [yurii@oc5337804250 stix-shifter]$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' results 'query1' 0 1
{'success': True, 'data': 'Results from search'}


(virtualenv) [yurii@oc5337804250 stix-shifter]$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' ping
{'success': True}
(virtualenv) [yurii@oc5337804250 stix-shifter]$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' query 'query1'
{'code': 200, 'query_id': 'uuid_1234567890'}
(virtualenv) [yurii@oc5337804250 stix-shifter]$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' status 'uuid_1234567890'
{'success': True, 'status': 'COMPLETED'}
(virtualenv) [yurii@oc5337804250 stix-shifter]$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' results 'uuid_1234567890' 0 1
{'success': True, 'data': 'Results from search'}

Reply:
I wonder it should be 
`{'code': 200, 'search_id': 'uuid_1234567890'}`
instead of 
`{'code': 200, 'query_id': 'uuid_1234567890'}` 
in async_dummy query call

Reply:
right,

$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' ping
{'success': True}
$ 
$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' query 'query1'
{'success': True, 'search_id': 'query1'}
$ 
$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' status 'query1'
{'success': True, 'status': 'COMPLETED', 'progress': 100}
$ 
$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' results 'query1' 0 1
{'success': True, 'data': 'Results from search'}
$ 
$ 
$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' ping
{'success': True}
$ 
$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' query 'query1'
{'search_id': 'uuid_1234567890'}
$ 
$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' status 'uuid_1234567890'
{'success': True, 'status': 'COMPLETED'}
$ 
$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' results 'uuid_1234567890' 0 1
{'success': True, 'data': 'Results from search'}